activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   For PostgreSQL >= 9.4 use `pgcrypto`'s `gen_random_uuid()` instead of
+    `uuid-ossp`'s UUID generation function.
+
+    *Yaw Boakye*
```

Comment:
The original of this PR is #21327. I think that it should be credited the original author.

```
    *Yuji Yaginuma*, *Yaw Boakye*
```